app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,61 @@
+require 'date'
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+    @task = Task.find_by(id: params[:id])
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    task = Task.new
+    task.name = params[:task][:name]
+    task.description = params[:task][:description]
+    task.complete = "not complete"
```

Comment:
This isn't working, I think you changed the schema.

To work, this would need to be
```ruby
task.completion_date = Time.now
```

Or, better yet, make a migration to remove `completion_date` and add `complete`